domain-cc/cc-app/src/python_src/api.py

```diff
@@ -93,7 +93,9 @@ def log_claim_stats(claim: Claim, classification: Optional[PredictedClassificati
     contention_text = claim.contention_text or ""
     is_in_dropdown = contention_text.strip().lower() in dropdown_values
     is_mapped_text = dropdown_lookup_table.get(contention_text, None) is not None
-    log_contention_text = contention_text if is_mapped_text else "unmapped contention text"
+    log_contention_text = (
+        contention_text if is_mapped_text else "unmapped contention text"
+    )
```

Comment:
Did you setup a new autoformatter?

Reply:
I didn't change formatting. Precommit hooks are defined [here](https://github.com/department-of-veterans-affairs/abd-vro/blob/a4003bf20f4e2558c704f96009203eb8b6386b8e/domain-cc/.pre-commit-config.yaml#L27). Perhaps I had committed w/ `--no-verify` prior, not sure